symfony/framework-bundle/3.3/bootstrap.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
Why not store the `bootstrap.php` file in the `src/` directory instead?

Reply:
because src/ is configurable, so it cannot be safely referenced from other recipes.